Sources/Utility/PkgConfig.swift

```diff
@@ -42,23 +96,18 @@ public struct PkgConfig {
     /// The list of libraries to link.
     public let libs: [String]
 
-    /// The built-in search path list.
-    ///
-    /// By default, this is combined with the search paths inferred from
-    /// `pkg-config` itself.
-    private static let searchPaths = [
-        AbsolutePath("/usr/local/lib/pkgconfig"),
-        AbsolutePath("/usr/local/share/pkgconfig"),
-        AbsolutePath("/usr/lib/pkgconfig"),
-        AbsolutePath("/usr/share/pkgconfig"),
-    ]
+    /// DiagnosticsEngine to emit warnings
+    public let diagnostics: DiagnosticsEngine
```

Comment:
This doesn't need to be public. I prefer using private unless any actually required.